app/config.js

```diff
@@ -56,8 +56,17 @@ module.exports = require('rc')('agregore', {
   hyperOptions: {
     storage: DEFAULT_HYPER_DIR
   },
-  // All options here: https://github.com/ssbc/ssb-config#configuration
-  ssbOptions: {},
+  /**
+   * All ssb options here: https://github.com/ssbc/ssb-config#configuration
+   * For bundled ssb server we use ssbd. Pass in an array of require'd ssb plugins
+   */
+  ssbOptions: {
+    appname: DEFAULT_SSB_APPNAME,
+    ssbd: {
+      runServer: true,
+      plugins: require('@metacentre/shipyard-ssb')
```

Comment:
Yeah, still feels like this should be a list of paths to require plugins from. 🤔

e.g. if you use `npm install` in some other folder, you could still add the plugin path inside your local node_modules.

Like

```
cd ~/
npm install my-plugin
```

Then have a config that looks like

{
  ssbOptions: {
    ssbd: {
      plugins: ["@metacentre/shipyard-ssb", "~/node_modules/my-plugin"]
    }
  }
}

Reply:
Whoops, totally missed the part of the docs which covered that you already implemented this. 😅